app/presenters/menu/default_menu.rb

```diff
@@ -245,6 +245,49 @@ def optimize_menu_section
         ])
       end
 
+      def alerts_overview_item
+        Menu::Item.new(
+          'monitor_alerts_overview',
+          N_('Overview'),
+          'monitor_alerts_overview',
+          {:feature => 'monitor_alerts_overview', :any => true},
+          '/alerts_overview'
+        )
+      end
+
+      def alerts_list_item
+        Menu::Item.new(
+          'monitor_alerts_list',
+          N_('All Alerts'),
+          'monitor_alerts_list',
+          {:feature => 'monitor_alerts_list', :any => true},
+          '/alerts_list'
+        )
+      end
+
+      def alerts_most_recent_item
+        Menu::Item.new(
+          'monitor_alerts_most_recent',
+          N_('Most Recent Alerts'),
+          'monitor_alerts_most_recent',
+          {:feature => 'monitor_alerts_most_recent', :any => true},
+          '/alerts_most_recent'
+        )
```

Comment:
I understand that.

We should probably disable the long line check for the menu definition and disable the alignment check for the toolbars.

Both seem much more readable when we don't follow the rules. And readability is the goal, not following Rubocop in all details.

There seems to be a way to disable a  check using comments in the source code -- see https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md and of the page.

That do you think?